src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -0,0 +1,322 @@
+        assert len(node.args) == 2
+        stencil_expr, domain_expr = node.args
+        # expect stencil (represented as a lambda function) as first argument
+        assert isinstance(stencil_expr, itir.Lambda)
+        # the domain of the field operator is passed as second argument
+        assert isinstance(domain_expr, itir.FunCall)
+
+        # add local storage to compute the field operator over the given domain
+        # TODO: use type inference to determine the result type
+        node_type = ts.ScalarType(kind=ts.ScalarKind.FLOAT64)
+        field_domain = dace_fieldview_util.get_domain(domain_expr)
+
+        # first visit the list of arguments and build a symbol map
+        stencil_args: list[gtir_to_tasklet.IteratorExpr | gtir_to_tasklet.MemletExpr] = []
+        for arg in self.callable_args:
+            fields: list[SDFGField] = sdfg_builder.visit(arg, sdfg=sdfg, head_state=state)
+            assert len(fields) == 1
+            data_node, arg_type = fields[0]
+            # require all argument nodes to be data access nodes (no symbols)
+            assert isinstance(data_node, dace.nodes.AccessNode)
+
+            if isinstance(arg_type, ts.ScalarType):
+                scalar_arg = gtir_to_tasklet.MemletExpr(data_node, sbs.Indices([0]))
+                stencil_args.append(scalar_arg)
+            else:
+                assert isinstance(arg_type, ts.FieldType)
+                indices: dict[Dimension, gtir_to_tasklet.IteratorIndexExpr] = {
+                    dim: gtir_to_tasklet.SymbolExpr(
+                        dace.symbolic.SymExpr(DIMENSION_INDEX_FMT.format(dim=dim.value)),
+                        ITERATOR_INDEX_DTYPE,
+                    )
+                    for dim, _, _ in field_domain
+                }
+                iterator_arg = gtir_to_tasklet.IteratorExpr(
+                    data_node,
+                    arg_type.dims,
+                    indices,
+                )
+                stencil_args.append(iterator_arg)
+
+        # represent the field operator as a mapped tasklet graph, which will range over the field domain
+        taskgen = gtir_to_tasklet.LambdaToTasklet(sdfg, state, sdfg_builder.offset_provider)
+        input_connections, output_expr = taskgen.visit(stencil_expr, args=stencil_args)
+        assert isinstance(output_expr, gtir_to_tasklet.ValueExpr)
+        output_desc = output_expr.node.desc(sdfg)
+
+        # retrieve the tasklet node which writes the result
+        last_node = state.in_edges(output_expr.node)[0].src
+        if isinstance(last_node, dace.nodes.Tasklet):
+            # the last transient node can be deleted
+            last_node_connector = state.in_edges(output_expr.node)[0].src_conn
+            state.remove_node(output_expr.node)
+        else:
+            last_node = output_expr.node
+            last_node_connector = None
+
+        # allocate local temporary storage for the result field
+        field_dims = [dim for dim, _, _ in field_domain]
```

Comment:
Just an idea since there are so many `_` in the following list comprehensions with `field_domain`, maybe unpacking its components first would help:
```python
        dims, ubs, lbs = zip(*field_domain)
```